sdk/python/assets/assets-in-registry/share-models-components-environments.ipynb

```diff
@@ -598,11 +601,6 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.8.8"
-  },
-  "vscode": {
-   "interpreter": {
```

Comment:
Is this delete expected?

Reply:
The tests passed with when this change came in so guessing should be okay.